src/mon/MonCommands.h

```diff
@@ -114,6 +114,9 @@ COMMAND("pg getmap", "get binary pg map to -o/stdout", "pg", "r", "cli,rest")
 COMMAND("pg send_pg_creates", "trigger pg creates to be issued",\
 	"pg", "rw", "cli,rest")
 COMMAND("pg dump " \
+        "name=poolid,type=CephInt,req=false " \
+        "name=osdid,type=CephOsdName,req=false " \
+	"name=states,type=CephChoices,strings=active|clean|down|replay|spliting|scrubbing|scrubq|degraded|inconsistent|peering|repair|recovery|backfill_wait|incomplete|stale|remapped|deep_scrub|backfill|backfill_toofull|recovery_wait|undersized,n=N,req=false " \
 	"name=dumpcontents,type=CephChoices,strings=all|summary|sum|delta|pools|osds|pgs|pgs_brief,n=N,req=false", \
```

Comment:
dumpcontents should be the first optional argument otherwise it will change the existing behavior